frontends/election-manager/src/components/confirm_removing_file_modal.tsx

```diff
@@ -28,27 +29,31 @@ export function ConfirmRemovingFileModal({
     ExternalTallySourceType.Manual
   );
 
+  const cvrFilesQuery = useCvrFilesQuery();
+
+  const isLoading =
+    (fileType === ResultsFileType.CastVoteRecord ||
+      fileType === ResultsFileType.All) &&
+    cvrFilesQuery.isLoading;
+
   let mainContent: React.ReactNode = null;
   let fileTypeName = '';
   let singleFileRemoval = true;
   switch (fileType) {
     case ResultsFileType.CastVoteRecord: {
-      const { fileList } = castVoteRecordFiles;
+      if (isLoading) {
+        return <Loading />;
+      }
+
+      const fileList = cvrFilesQuery.data || [];
```

Comment:
Do you still need a default value here to get around `data` possibly being undefined? If so, that's probably because checking `query.isLoading` isn't sufficient, we also need to check `query.isError` (or, use `query.isSuccess` to check both). Even if we're going to ignore errors, I think it would be good to do it explicitly.

Reply:
(And noting there are a few other places with similar patterns)